Library/ViewModels/PledgeViewModel.swift

```diff
@@ -867,6 +882,7 @@ private func checkoutPropertiesData(from createBackingData: CreateBackingData, i
 
   return Koala.CheckoutPropertiesData(
     amount: amount,
+    checkoutId: checkoutId,
```

Comment:
While testing this I noticed that the `checkoutId` that's returned from Graph is the `relayId`. I think for tracking we actually want to send the `Int` value of the `checkoutId` 🤔